option optimize_for = LITE_RUNTIME;

message LegionChangeBulletinRequest
{
 required string content = 1;
}
